src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -378,7 +378,9 @@ protected function prepareException(Throwable $e)
         return match (true) {
             $e instanceof BackedEnumCaseNotFoundException => new NotFoundHttpException($e->getMessage(), $e),
             $e instanceof ModelNotFoundException => new NotFoundHttpException($e->getMessage(), $e),
-            $e instanceof AuthorizationException && $e->hasStatus() => new HttpException($e->status(), $e->getMessage(), $e),
+            $e instanceof AuthorizationException && $e->hasStatus() => new HttpException(
+                $e->status(), $e->response()?->message() ?: (Response::$statusTexts[$e->status()] ?? 'Whoops, looks like something went wrong.'), $e
+            ),
```

Comment:
Note that `'Whoops, looks like something went wrong.'` is only presented to the user when:

1. The deny action did not receive a message.
2. The status code in unknown to Symfony.

Also, this message is really on for JSON responses, as Symfony will show this message no matter what we change this too. For example:

```php
// .env:

APP_DEBUG=false


// Route:

Route::get('/', function () {
    throw new HttpException(399, 'foo');
});
```

<img width="684" alt="Screen Shot 2022-08-31 at 10 12 48 am" src="https://user-images.githubusercontent.com/24803032/187565337-8bbe856d-cae3-42e0-af07-95831fea1838.png">

So using this messaging actually creates consistency between the HTML response and the JSON response.